test/apps/sockaddr/sa_tcp.cc

```diff
@@ -4,10 +4,16 @@
  * See file LICENSE for terms.
  */
 
+#ifdef HAVE_CONFIG_H
```

Comment:
maybe move the inclusion of epoll.h to sa_util.h, instead of putting it in multiple files in test/apps/sockaddr?
also, how would the code compile if epoll is not defined?

Reply:
I do not like idea of pulling conditionally-included headers into headers, because this means that the including header now depends on config.h.

For now, I just want to conditionally disable the header use, port to kqueue (vs. use of epoll-shims) is for the next stage.

Reply:
but just disabling the headers won't make it compile right? since the code would still try to use epoll function calls?

Reply:
Of course, but it makes the issues visible.  Otherwise, either gcc or clang refuse to compile the file with missed headers at all.

Reply:
ok